CommonMark/CommonMark.py

```diff
@@ -1078,8 +1078,7 @@ def closeUnmatchedBlocks(self, already_done, oldtip):
         if blank and container.last_line_blank:
             self.breakOutOfLists(container, line_number)
 
-        while container.t != "ExtensionBlock" and \
```

Comment:
For now, maybe a better solution is to just remove those failing tests from the spec, because it sounds like we shouldn't revert what you've added. I'll update this PR.